index.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   config: function(env, baseConfig) {
     var config = this._super.config.apply(this, arguments);
 
+    baseConfig.APP = baseConfig.APP || {};
```

Comment:
can we instead check if APP exists, and if not just do nothing?

Reply:
done!